### PR TITLE
user/nxengine-evo: new package

### PR DESCRIPTION
**Pull request by tulilirockz**
just had to get the data/ directory from the windows release cuz the source code does not contain the necessary assets to actually boot up the game

### CODE REVIEW

File: user/nxengine-evo/template.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+pkgname = "nxengine-evo"
+pkgver = "2.6.5"
```
Comment (nekopsykose) [inline user/nxengine-evo/template.py:2]:
```suggestion
pkgver = "2.6.5.1"
```

**Comment by nekopsykose** [comment]
this has ancient vendored copies of utfcpp/spdlog in deps/ that will fail to compile sooner or later

**Comment by tulilirockz** [comment]
managed to forcefully link the system libraries and i changed the include_path to have the necessary deps in it. is this fine or is a general patch that changes the "include <json.hpp>"-like declarations to have the proper include path preferrable? - its just that it would be a kind of big patch

**Comment by nekopsykose** [comment]
dw, did it for u

**Pull request merged**